.cicd/helpers/general.sh

```diff
@@ -0,0 +1,7 @@
+export ROOT_DIR=$( dirname "${BASH_SOURCE[0]}" )/../..
+export BUILD_DIR=$ROOT_DIR/build
+export CICD_DIR=$ROOT_DIR/.cicd
+export HELPERS_DIR=$CICD_DIR/helpers
+export JOBS=${JOBS:-"$(getconf _NPROCESSORS_ONLN)"}
+
+. $HELPERS_DIR/logging.sh
```

Comment:
I still think we should get rid of this whole file in favor of hard-coding these paths, because we expect they will never change. Hiding this off in a separate file makes things too obtuse. Even opening this file, it wouldn't be obvious to our customers where these paths are. Let's simplify this and make it more user friendly.